webui/react/src/hooks/useModal/useModalExperimentCreate.tsx

```diff
@@ -141,7 +141,8 @@ const useModalExperimentCreate = (props: Props): ModalHooks => {
   }, []);
 
   const handleCancel = useCallback((close) => {
-    if (close?.triggerCancel) {
+    // 'close' is an indicator for if cancel button (show config) is clicked or not.
```

Comment:
Currently `close` is typed as `any`. Can we figure out what type it should actually be and specify that?